downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -62,6 +62,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 #### CLI
 
+##### Auto restarting Ballerina Services
```

Comment:
```suggestion
* Introduced auto-restarting of services with `bal run` command as an experimental feature.
```bash
$ bal run --watch
```

Reply:
Remove `bash` in line 67. It will render the command as a bash script which results in incorrect syntax highlighting.

Reply:
Changed with fa8276b5b552f3c8858aa2c7cb9f27e24fe25b48